internal/random/random.go

```diff
@@ -15,9 +15,9 @@ var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
 
 func randSeq(n int) string {
 	b := make([]rune, n)
-	rand.Seed(time.Now().UnixNano())
+	rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
If this is merged after https://github.com/cloudquery/plugin-sdk/pull/1255 the diff goes away